requirements.txt

```diff
@@ -4,3 +4,5 @@ google-api-python-client
 google-auth
 google-auth-httplib2
 google-auth-oauthlib
+google-cloud
```

Comment:
while this is fine, I _think_ this could be `google-cloud-bigquery`, which is smaller?

Reply:
:+1:  to using `google-cloud-bigquery` instead of `google-cloud`

Reply:
Verified on a clean environment that `google-cloud-bigquery==0.25.0` is enough. However, `0.26.0` throws the following error:

```
  File "gbq.py", line 960, in read_gbq
    query_results = query_job.results()
AttributeError: 'QueryJob' object has no attribute 'results'
```

Pinning to `0.25.0` for now, thanks, great suggestion.

Reply:
Thanks for checking @jasonqng 

Although that's a bit concerning that this breaks for recent versions of that library - that's going to force people to roll back upgrades. Others can come to a verdict on whether that's a showstopper, but at least worth seeing if we can make it compatible with a newer version?

Reply:
You'll also need to pin the version in `setup.py`.

Reply:
@MaximilianR Let me see if I can figure out how to unbreak it for 0.26. If so, I'll catch any exceptions and handle. But otherwise, hopefully there is enough value to be gained that users would be willing to roll back to 0.25, if not in their daily usage, then at least in a dedicated environment when they want to use pandas-gbq in its current state, at least until we figure out a way to support future stable versions of google-cloud-bigquery

Reply:
@MaximilianR Figured out why it broke. Works now for latest (0.26.0).